src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -640,31 +644,50 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   /**
    * Handles update events from address sub-components.
    */
-  protected updateAddress (addressToUpdate: AddressTypes, baseAddress: AddressIF, newAddress: AddressIF): void {
-    Object.assign(baseAddress, newAddress)
+  protected updateAddress (addressToUpdate: AddressTypes, newAddress: AddressIF): void {
     switch (addressToUpdate) {
       case AddressTypes.MAILING_ADDRESS:
-        if (this.inheritMailingAddress) {
-          this.deliveryAddress = { ...newAddress, addressType: 'delivery', id: this.deliveryAddress.id }
-        }
-        if (this.inheritRegisteredAddress) {
-          this.recMailingAddress = { ...newAddress, addressType: 'mailing', id: this.recMailingAddress.id }
-          this.recDeliveryAddress = { ...this.deliveryAddress, addressType: 'delivery', id: this.recDeliveryAddress.id }
+        // only update if not equal
+        if (!isEqual(this.mailingAddress, newAddress)) {
+          this.mailingAddress = { ...newAddress }
```

Comment:
Checking if not equal is the key to preventing a prop-event update infinite loop.

The problem: this assignment causes the BaseAddress component to refresh itself and emit an update event, which would cause this assignment, etc.

The solution: don't reassign if the updated address is the same as the current address.

As for reactivity, yes, that is why the BaseAddress component refreshes itself. It is also why any getters that use this property (eg, `disableSameDeliveryAddress`) refresh themselves. (Getter are otherwise cached.)